src/personalnotification.cpp

```diff
@@ -158,10 +158,12 @@ void PersonalNotification::publishNotification()
         connect(m_notification, SIGNAL(closed(uint)), SLOT(onClosed(uint)));
 
         m_notification->setTimestamp(QDateTime::currentDateTimeUtc());
+        m_notification->setHintValue("x-nemo-feedback", "sms,sms_exists");
+    } else {
+        m_notification->setHintValue("x-nemo-feedback", "sms_exists");
```

Comment:
- The event type here can be something else than sms too, thus should map to feedback names accordingly? 
- The groupType() function seems getting unused. 
- If the usage for the category files is removed, the related category files should be also removed.
- If not using the categories, should apply the other notification properties set there. E.g. the icon, display on and priority. Transient I guess not needed if there's only one notification instead of separate for popup and on the events view.

Reply:
Oops, sorry, forgot git add. But good point about the other category options.